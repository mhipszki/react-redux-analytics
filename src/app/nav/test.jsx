import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import Nav from './';

test('styles a button element', () => {
  const component = shallow(<Nav />);
  expect(component).toMatchSnapshot();
});

test('has different styling when disabled', () => {
  const component = shallow(<Nav disabled />);
  expect(component).toMatchSnapshot();
});
