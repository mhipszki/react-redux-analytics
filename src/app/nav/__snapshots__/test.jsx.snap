// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has different styling when disabled 1`] = `
.c0 {
  background: #eee;
  border: 2px solid #eee;
  border-radius: 3px;
  color: white;
  cursor: pointer;
  font-size: 1.5em;
  margin: 0 1em;
  padding: 0.25em 1em;
}

<button
  className="c0"
  disabled={true}
/>
`;

exports[`styles a button element 1`] = `
.c0 {
  background: palevioletred;
  border: 2px solid palevioletred;
  border-radius: 3px;
  color: white;
  cursor: pointer;
  font-size: 1.5em;
  margin: 0 1em;
  padding: 0.25em 1em;
}

<button
  className="c0"
/>
`;
